src/documents/APIChangesFrom1_5To2_0.md

```diff
@@ -0,0 +1,136 @@
+---
+title: API Changes from 1.5 to 2.0
+layout: default
+---
+
+# API Changes from 1.5 to 2.0 #
+
+## Framework Changes ##
+
+### Preferences Framework ###
+
+#### Namespace Changes ####
+
+Rename "fluid.prefs.enactors" to "fluid.prefs.enactor"
+
+#### Component Grade Changes ####
```

Comment:
yes and yes. Basically the grades and model sharing can be used by anything. I think they were actually both in the framework for Infusion 1.5, if I'm not mistaken. However, now we are actually making use of these in the preferences framework.